Objects/longobject.c

```diff
@@ -130,22 +135,34 @@ long_normalize(PyLongObject *v)
 PyLongObject *
 _PyLong_New(Py_ssize_t size)
 {
+    struct _Py_long_state *state = &_PyInterpreterState_GET()->long_state;
+#ifdef Py_DEBUG
+    // _PyLong_New() must not be called after _PyFloat_Fini()
```

Comment:
Did you mean `_PyLong_Fini()`?
```suggestion
    // _PyLong_New() must not be called after _PyLong_Fini()
```